digital_asset_types/src/dao/scopes/asset.rs

```diff
@@ -83,7 +83,8 @@ pub async fn get_by_grouping(
 ) -> Result<Vec<FullAsset>, DbErr> {
     let condition = asset_grouping::Column::GroupKey
         .eq(group_key)
-        .and(asset_grouping::Column::GroupValue.eq(group_value));
+        .and(asset_grouping::Column::GroupValue.eq(group_value))
+        .and(asset_grouping::Column::Verified.eq(true));
```

Comment:
is this condition strictly required? seems it changes the query a bit ?

Reply:
So the way it used to be was that stuff was ONLY in the `asset_grouping` table if it was `verified`.  Because of that, minting with an unverified collection and verifying later would not index properly (had nothing to do with order of ingestion. The ingester just ignored the instructions).

So adding the verified flag is needed to make the indexing work, and changing the query makes it consistent with how it worked before.  It will not return an unverified collection, just as before.

Reply:
I will add this check for others as well.  We can change the API to instead return the verified value as creators are returned, or we can add a bool flag for the caller to request verified only.  But if so, I think that should be a follow on PR to discuss specifically.

The reason I don't want to automatically include unverified collections is that it could lead to issues with marketplaces listing something that is not truly part of a collection since anyone could set their collection to anything without it being verified.